tests/visualization_tests/test_common.py

```diff
@@ -0,0 +1,44 @@
+import plotly.express as px
```

Comment:
It would be natural to move the contents of this test file to the respective visualization function test file and delete this file. The content of the test itself is slightly different for each test function, and there are still a few lines of test duplicated among the test files of the visualization functions. E,g, `test_target_is_none_and_study_is_multi_obj` in `test_parallel_coordinate.py` and `test_slice.py`.

Reply:
But `test_same_color_map ` in this test file aims to check whether or not to use colormap *among different plot functions*.

Reply:
Ah, okay, we can delete the test file by updating the pre-defined colourmap: `COLOR_SCALE=px.colors.sequential.Blues` in `utils.py`, and using this `COLOR_SCALE` in each test file.